amethyst_rendy/src/pass/debug_lines.rs

```diff
@@ -222,7 +222,7 @@ fn build_lines_pipeline<B: Backend>(
                     blend: Some(pso::BlendState::ALPHA),
                 }])
                 .with_depth_test(pso::DepthTest {
-                    fun: pso::Comparison::LessEqual,
```

Comment:
I'm a bit worried about this one. The skybox had LessEqual for good reason, but I fail to see why it was like this here.

GreaterEqual would be useful if debuglines was somehow modified to sometimes draw lines at infinity.

Reply:
Changed it to GreaterEqual. That causes later lines to be drawn on top of earlier ones. It should also make them visible in a 2d game.